CMakeLists.txt

```diff
@@ -580,6 +580,8 @@ if(MSVC)
   add_definitions(-D_SILENCE_ALL_CXX17_DEPRECATION_WARNINGS)
   add_definitions(-D_SILENCE_FPOS_SEEKPOS_DEPRECATION_WARNING)
 
+  add_compile_options(/utf-8)
```

Comment:
https://docs.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-and-executable-character-sets-to-utf-8?view=msvc-170

This avoids the gibberish when print box characters in the Labs cli.